api/src/services/graphql.ts

```diff
@@ -1578,6 +1578,7 @@ export class GraphQLService {
 		{
 			CreateCollectionTypes,
 			ReadCollectionTypes,
+			UpdateCollectionTypes,
 			DeleteCollectionTypes,
 		}: {
```

Comment:
I'm a little concerned about some kind of unknown (to me) cascading effect from this. Any concerns?

Reply:
That's a question for the actual engineering team 😉

Reply:
No side effects 👍🏻